BDArmory/Weapons/Missiles/BDModularGuidance.cs

```diff
@@ -1032,6 +1032,12 @@ public void ExecuteNextStage()
             vessel.ActionGroups.ToggleGroup(
                 (KSPActionGroup)Enum.Parse(typeof(KSPActionGroup), "Custom0" + (int)_nextStage));
 
+            if (StagesNumber == 1)
+            {
+                if (SpawnUtils.CountActiveEngines(vessel) < 1)
+                    SpawnUtils.ActivateAllEngines(vessel);
```

Comment:
`ActivateAllEngines` will need to have the `ignoreModularMissileEngines` parameter being `false` here.